README.md

```diff
@@ -123,27 +123,25 @@ The below activities practice understanding how the components interact with eac
 
 ### Rendering Views
 
-**The structure of our HTML has hierarchy** When an HTML element is nested inside of another element, the outer-most element has a higher hierarchy. That outer-element acts as a container for it's inner-elements.
+**The structure of our HTML has hierarchy** All HTML elements are nested inside of **one** other element, other than the `<HTML>` tag. The *parent* element acts as a container for it's nested *child* elements.
 
-Some backbone views will be responsible for other views and act as a container with hierarchy over the inner-elements that are generated from the views.
+Backbone views can have the same relationship, and generally mirror the HTML's hierarchy. Some backbone views will be responsible for *child* views and act as a container with hierarchy over the inner-elements that are generated from those *child* views. For example, in the Rolodex project, the rolodex view was responsible for rendering many contact views.
 
-For example, in the Rolodex project, the Rolodex view was responsible for rendering many contact views
-
-
-With our application, a single page will likely have many different Backbone views rendering as once.
+With our Backbone applications, a single page will likely have many different Backbone views rendering at once.
 
 
 **Activity:** Views Responsibility
 
-With this activity, we will get more comfortable recognizing the code that is
-
-Each of the following views should correspond to a color. ex: trip view is yellow, trip_lis view is blue and app view is red. Use that color to draw a square around all the code that corresponds to that view, including view files, html
+With this activity, we will recognize what sections of the DOM map to specific views.
 
+Each of the following views should correspond to a color (or other distinct marking, like dashed lines).
+**ex:** tripView is yellow, tripListView is blue and appView is red.
 
-- trip view
-- trip_list view
-- app view
+On the worksheet, use that color to draw a square around all the code snippets that correspond to each of these views:
 
+- tripView
+- tripListView
+- appView
 
```

Comment:
Do you want to add something towards the end like - once you are done, compare with your neighbor and review the code together to come to a consesus?